src/Settings/Default.hs

```diff
@@ -105,7 +105,8 @@ defaultPackages = mconcat
 packagesStage0 :: Packages
 packagesStage0 = mconcat
     [ append [ binary, cabal, compiler, ghc, ghcBoot, ghcBootTh, ghcCabal
-             , ghcPkg, hsc2hs, hoopl, hpc, templateHaskell, transformers ]
+             , ghcPkg, hsc2hs, hoopl, hpc, templateHaskell, transformers
+             , mkUserGuidePart ]
     -- the stage0 predicate makes sure these packages are built only in Stage0
     , stage0 ? append [ deriveConstants, dllSplit, genapply, genprimopcode
                       , hp2ps, unlit ]
```

Comment:
You need to add `mkUserGuidePart` here. The above list actually gets built both in `Stage0` and in `Stage1`.